google/extensions.rb

```diff
@@ -18,14 +18,6 @@ def underscore
     Google::StringUtils.underscore(self)
   end
 
-  def camelize(style = :lower)
```

Comment:
`camelize` is coming out of active support now?

Reply:
It has been ever since we started using ActiveSupport in the first place, I believe. This specific change was a no-op.